recipes/deployment.rb

```diff
@@ -11,13 +11,13 @@
   if prefer :database, 'sqlite'
     gsub_file 'Gemfile', /.*gem 'sqlite3'\n/, ''
     add_gem 'sqlite3', group: [:development, :test]
-    add_gem 'pg', group: :production
+    add_gem 'pg', '>= 0.18', group: :production
   end
 end
 
 if prefer :deployment, 'capistrano3'
   say_wizard "installing gems for Capistrano"
-  add_gem 'capistrano', '~> 3.0.1', group: :development
+  add_gem 'capistrano', '>= 3.6.0', group: :development
```

Comment:
@MProG  `>= 3.6` says that we can update this gem to any version that will be bigger than `3.6`

what do you think about specifying major version here
```  
gem 'capistrano', '~> 3.6.0'
```
in order not to update it to `3.7` or later. updating to later version should be done by hands for such important gems